src/promptflow/promptflow/_sdk/_service/apis/flow.py

```diff
@@ -0,0 +1,119 @@
+    @api.doc(description="Flow test")
+    @api.expect(flow_test_model)
+    def post(self):
+        flow = api.payload["flow"]
+        inputs = api.payload.get("inputs", None)
+        environment_variables = api.payload.get("environment_variables", None)
+        variant = api.payload.get("variant", None)
+        node = api.payload.get("node", None)
+        experiment = api.payload.get("experiment", None)
+        output_path = api.payload.get("output_path", None)
+        if Configuration.get_instance().is_internal_features_enabled() and experiment:
+            result = get_client_from_request().flows.test(
+                flow=flow,
+                inputs=inputs,
+                environment_variables=environment_variables,
+                variant=variant,
+                node=node,
+                experiment=experiment,
+                output_path=output_path,
+            )
+        else:
+            result = get_client_from_request().flows.test(
+                flow=flow,
+                inputs=inputs,
+                environment_variables=environment_variables,
+                variant=variant,
+                node=node,
+                allow_generator_output=False,
+                stream_output=False,
+                dump_test_result=True,
+                output_path=output_path,
+            )
+        return result
+
+
+@api.route("/get")
+class FlowGet(Resource):
+    @api.response(code=200, description="Return flow yaml as json", model=dict_field)
+    @api.doc(description="Return flow yaml as json")
+    def get(self):
+        flow_path = api.payload["flow"]
+        flow_path = resolve_flow_path(Path(flow_path))
+        flow_info = load_yaml(flow_path)
+        return flow_info
+
+
+@api.route("/ux_inputs")
+class FlowUxInputs(Resource):
+    @api.response(code=200, description="Get the file content of file UX_INPUTS_JSON", model=dict_field)
+    @api.doc(description="Get the file content of file UX_INPUTS_JSON")
+    @api.expect(flow_ux_input_model)
+    def get(self):
+        flow_path = api.payload["flow"]
+        flow_ux_inputs_path = Path(flow_path) / PROMPT_FLOW_DIR_NAME / UX_INPUTS_JSON
+        if not flow_ux_inputs_path.exists():
+            flow_ux_inputs_path.touch(mode=read_write_by_user(), exist_ok=True)
+        try:
+            ux_inputs = json_load(flow_ux_inputs_path)
+        except json.decoder.JSONDecodeError:
+            ux_inputs = {}
+        return ux_inputs
+
+    @api.response(code=200, description="Set the file content of file UX_INPUTS_JSON", model=dict_field)
+    @api.doc(description="Set the file content of file UX_INPUTS_JSON")
+    @api.expect(flow_ux_input_model)
+    def post(self):
+        content = request.get_json()
+        flow_path = api.payload["flow"]
+        flow_ux_inputs_path = Path(flow_path) / PROMPT_FLOW_DIR_NAME / UX_INPUTS_JSON
+        flow_ux_inputs_path.touch(mode=read_write_by_user(), exist_ok=True)
+        json_dump(content, UX_INPUTS_JSON)
```

Comment:
Should be `json_dump(content, flow_ux_inputs_path)`. And I prefer to add `indent: 2` to make the json file content pretty.